argo-cd-apps/overlays/development/repo-overlay.yaml

```diff
@@ -41,3 +41,14 @@ spec:
   source:
     repoURL: https://github.com/redhat-appstudio/infra-deployments.git
     targetRevision: main
+---
+apiVersion: argoproj.io/v1alpha1
```

Comment:
Wondering how it's going to be used/tested in development mode?

Reply:
I just had that there to test with my fork. I guess it's not needed.

Reply:
No it's fine, if you've found it useful, I'd be curious how you've used it, then we could have devs use this as well.

Reply:
It was useful to test the pipeline runs. I submitted a PR to my fork as the test case. The only pain point was adding the GitHub app secret, app id and route on each deployment. I can create a script to make that easier though.

Reply:
Hi @sbose78 here's my thoughts on testing it in dev mode. The commit message has a description of the usage. https://github.com/redhat-appstudio/infra-deployments/pull/126/commits/4ef258b694d283b5a9b5b4c28d889da0faf466b8